packages/components/src/Modal/ConfirmationModal/ConfirmationModal.module.scss

```diff
@@ -35,14 +34,87 @@ $dt-assertive-header-background: $color-orange-100;
 
   // override Murmur global styles :(
   h1 {
-    color: $dt-heading-text-color;
+    color: var(color-purple-800);
   }
 }
 
 .iconContainer {
   position: relative;
+  align-self: flex-start;
+
+  @media (min-width: $layout-breakpoints-medium) {
+    align-self: center;
+  }
+
+  .spotIcon {
+    width: 30px;
+    height: 30px;
+
+    @media (max-width: $layout-breakpoints-medium) {
+      margin-right: 0.9375rem;
+      position: relative;
+      top: 0.125rem;
+    }
+  }
 }
 
+.prominent {
+  display: grid;
+  grid-template-columns: 1.1fr 2fr;
+  align-items: center;
+
+  @media (max-width: $layout-breakpoints-medium) {
+    grid-template-columns: 1fr;
+    display: flex;
+    flex-direction: column;
+    place-content: flex-start center;
+    align-items: flex-start;
+  }
+
+  .iconContainer {
+    position: absolute;
+    top: 8%;
+
+    @media (max-width: $layout-breakpoints-medium) {
+      position: relative;
+      top: 0;
+      margin-bottom: $spacing-sm;
+    }
+  }
+
+  .spotIcon {
+    width: 155px;
+    height: 155px;
+    margin: 0 auto;
+    color: $color-purple-800;
+
+    @media (max-width: $layout-breakpoints-medium) {
+      width: 86px;
+      height: 86px;
+    }
+  }
+}
+
+.padded {
+  padding: $spacing-md $spacing-lg $spacing-md $spacing-lg;
+
+  @media (max-width: $layout-breakpoints-medium) {
+    padding: $spacing-md;
+  }
+}
+
+// --------------------------------
+// @deprecated moods use variant styles instead
+// --------------------------------
+
+$dt-positive-header-background: $color-green-100;
```

Comment:
Just confirming we're only keeping these because they'll be axed in the next major?

Reply:
Yep, on the chopping block